src/js/common/schemaform/widgets/SSNWidget.jsx

```diff
@@ -11,7 +11,11 @@ export default class SSNWidget extends React.Component {
     this.state = { val: props.value };
   }
   handleChange = (val) => {
-    const strippedSSN = val.replace(/[\- ]/g, '');
+    let strippedSSN;
```

Comment:
Might put a comment in here about intentionally letting `undefined` through. It sort of looks like we're accidentally (by not setting strippedSSN when val is falsy) allowing onChange to be called with `undefined`.